docs/calendar.md

```diff
@@ -0,0 +1,277 @@
+# Temporal.Calendar
+
+<details>
+  <summary><strong>Table of Contents</strong></summary>
+<!-- toc -->
+</details>
+
+A `Temporal.Calendar` is a representation of a calendar system.
+It includes information about how many days are in each year, how many months are in each year, how many days are in each month, and how to do arithmetic in that calendar system.
+
+Much of the world uses the [Gregorian calendar](https://en.wikipedia.org/wiki/Gregorian_calendar), which was invented in 1582 C.E.
+On the modern Internet, the most often used calendar system is the calendar standardized by ISO 8601, which is the same as the Gregorian calendar with the addition of week-numbering rules.
+In general it is extended backwards ("proleptically") to cover the period of history before its invention, which is an optional modification allowed by the ISO 8601 standard.
+
+However, the ISO 8601 calendar is not the only calendar in common use in the world.
+Some places use another calendar system as the main calendar, or have a separate calendar system as a commonly-used civil or religious calendar.
+
+> **NOTE:** The Temporal polyfill currently only includes the ISO 8601 calendar, but the Temporal proposal will eventually provide all the calendars supported by Intl.
+> See [issue #541](https://github.com/tc39/proposal-temporal/issues/541).
+
+### When to use the `Temporal.Calendar`
+
+Most code will not have to use the `Temporal.Calendar` API.
```

Comment:
Try to avoid statements such as "most code" can ignore non-ISO calendar systems.  That gives off the wrong impression.  Focus on when you need to use it, not when you don't need to use it.

Suggested re-write of these two paragraphs:

---

It is best practice to specify a calendar system when performing calendar-sensitive operations, which are those involving arithmetic or other calculation in months or years.

For example, to add a month to a date in the Hebrew calendar:

```javascript
date.withCalendar('hebrew').plus({ months: 1 })
```

Temporal's `.toLocaleString()` functions use the user's preferred calendar, without needing to call `.withCalendar()`.  To perform arithmetic consistently with the `.toLocaleString()` calendar system:

```javascript
const calendar = new Intl.DateTimeFormat().resolvedOptions().calendar;
date.withCalendar(calendar).plus({ months: 1 })
```

Reply:
Fixed in #636, will rebase.